javaslang-benchmark/src/test/java/javaslang/collection/VectorBenchmark.java

```diff
@@ -399,6 +399,20 @@ public Object scala_persistent() {
         }
 
         @Benchmark
+        public Object clojure_persistent() {
```

Comment:
Clojure's Vector doesn't have an prepend, so I'm reconstructing everything every time to simulate a prepend